src/__tests__/downshift.get-input-props.js

```diff
@@ -583,6 +583,25 @@ test(`getInputProps doesn't include event handlers when disabled is passed (for
   }
 })
 
+test('highlightedIndex is reset to defaultHighlightedIndex when inputValue changes', () => {
+  const props = {defaultHighlightedIndex: 0}
```

Comment:
maybe add `const defaultHighlightedIndex = 0` and use that in props and in assertion. Not a big deal but it can be better.

Reply:
That's fair